libvast/test/type.cpp

```diff
@@ -714,8 +714,7 @@ TEST(enriched types) {
   CHECK_EQUAL(aat.attribute("second"), "");
   CHECK_EQUAL(aat.attribute("third"), "nestingworks");
   CHECK_EQUAL(aat.attribute("fourth"), std::nullopt);
-  CHECK_EQUAL(fmt::format("{}", aat), "l2 #third=nestingworks #first=value "
-                                      "#second");
+  CHECK_EQUAL(fmt::format("{}", aat), "l2 #third=nestingworks");
```

Comment:
I would expect the nested at part to be displayed as well. But this is just my naive intuition. Can you elaborate a little bit why this is the expected behaviour?

Reply:
It was only just changed in this PR to align how we print and parse types. The nested attributes are part of the nested type already, so printing them again just sounds confusing. What we could do is add an option for the formatter to print the whole type structure instead of just the first layer.

Reply:
Alright, if we find it useful to have whole thing printed then we can implement it